ci/php.ini

```diff
@@ -1,12 +1,28 @@
 [PHP]
 ; Prod settings -- see ../prod/php.ini
 date.timezone = UTC
-zend.assertions	= 0
-upload_max_filesize = 24M
-post_max_size = 32M
 file_uploads = On
+display_errors = Off
+display_startup_errors = Off
```

Comment:
This is very disruptive for both our CI (where we may want to see a stacktrace in a screenshot) or local development.

Now that we change these values for the prod baseline we should ensure that we overwrite them in the CI/dev specific sections.

Reply:
I will change this.